model/src/util.py

```diff
@@ -5,6 +5,8 @@
 import snntorch as snn
```

Comment:
may not need this import...

Reply:
yeah there are a lot of imports I need to clean up as mypy doesn't catch it. Will address.